src/main/webapp/app/lecture/manage/lecture-units/attachment-video-unit-form/attachment-video-unit-form.component.ts

```diff
@@ -242,4 +289,8 @@ export class AttachmentVideoUnitFormComponent implements OnChanges {
     cancelForm() {
         this.onCancel.emit();
     }
+
+    shouldShowTranscriptionCreation(): boolean {
+        return this.accountService.isAdmin();
+    }
```

Comment:
You could change this to a computed signal, something like for example:

```suggestion
    readonly shouldShowTranscriptionCreation = computed(() => this.accountService.isAdmin());
```